rest-fights/src/main/resources/application.properties

```diff
@@ -14,15 +14,26 @@ quarkus.http.test-port=0
 ## Mongo
 quarkus.mongodb.metrics.enabled=true
 quarkus.mongodb.database=fights
-
+#quarkus.mongodb.connection-string=mongodb://localhost:52003
+#quarkus.mongodb.credentials.username=superfight
+#quarkus.mongodb.credentials.password=superfight
 ## Liquibase Mongo
 quarkus.liquibase-mongodb.migrate-at-start=true
 
 ## CORS
 quarkus.http.cors=true
-quarkus.http.cors.origins=*
-
+quarkus.http.cors.origins=http://localhost:8082,http://localhost:8080,http://localhost:8086
+quarkus.http.cors.access-control-allow-credentials=true
+org.eclipse.microprofile.rest.client.propagateHeaders=Cookie
```

Comment:
This probably shouldn't be set because it takes effect globally for all of the rest clients. You only want to set it for the auth-service client.

Reply:
can `org.eclipse.microprofile.rest.client.propagateHeaders` be set for individual clients? It wasn't mentioned in the Quarkus docs for the rest client and didn't see any mention in the microprofile docs

Reply:
Thats a good question that I don't know the answer to. @geoand do you know?

Basically be able to specify a header to be propagated over the resteasy reactive rest client but only for 1 specific client, not all rest clients that are defined within an app.

Reply:
I think you would have to create an implementation of `ReactiveClientHeadersFactory` and register it like in https://quarkus.io/guides/rest-client#custom-headers-support

In there you'd have access to all of the incoming headers and could selectively propagate headers.

Reply:
sounds good, I'll look into this